txpipe/theory.py

```diff
@@ -73,7 +73,12 @@ def get_ccl_tracers(self, s, cosmo, smooth=False):
             nz = smooth_nz(Ti.nz) if smooth else Ti.nz
             print("smooth:", smooth)
             # Convert to CCL form
-            tracers[name] = pyccl.WeakLensingTracer(cosmo, (Ti.z, nz))
+            try:
+                tracers[name] = pyccl.WeakLensingTracer(cosmo, (Ti.z, nz))
+            except:
```

Comment:
again, good to just catch the one error

Reply:
How can I specify the error? I tried it but I was not sure how to do it. Is it using NameError?